### ISSUE TITLE
DecapCMS feasibility analysis

### ISSUE THREAD

**Issue by futuresoup** [climate-tech-handbook/climate-tech-handbook#23]
We need to get a real assessment on  where the major holes are in our new CMS capability, and how easy/hard they will be to implement a fix.

**For example:**

- Is there a somewhat decent collaboration workflow out there? What would it take to implement a workaround and move towards a native solution?

- Are standard permissions controls just not there?
I’ll put together a ticket for that task soon.

**Comment by alexmayberry** [comment]
Love it. It would also be great to find a way to create super easy linking to other md pages directly from the CMS editor.

**Comment by alexmayberry** [comment]
@futuresoup what kind of collaboration features are you looking for? [Commenting like this seems really nice](https://spinalcms.com/alternative-to-decap-cms/) <-- that could even be linked to PR conversation/activity threads since [each saved edit is a commit to a PR](https://github.com/climate-tech-handbook/climate-tech-handbook/pull/19)

**Comment by futuresoup** [comment]
@alexmayberry yes, the [SpinalCMS collab workflow](https://spinalcms.com/alternative-to-decap-cms/) you found is a good step and worth exploring.

I tried SpinalCMS but the pages didn't work because of the missing front matter an issue finding the img folder, so images didn't load. I didn't have time to debug, so the 7 day trial ran out before I could test anything.
